### PR TITLE
DOC: Use plot_event in an example

### LINKED ISSUE #2775: Use plot_event in a example
The function `plot_event` has currently no example linked to its [doc](https://nilearn.github.io/modules/generated/nilearn.plotting.plot_event.html#nilearn.plotting.plot_event). 
It wouldn't be too costly to use it in one example somewhere.

### PR DESCRIPTION
**Pull request by Craskermasker**
Closes #2775 .

Changes proposed in this pull request:

- Optionally visualize events at end of this example

### CODE REVIEW

**Review by NicolasGensollen — COMMENTED** [review]
LGTM, thanks @Craskermasker !
Could you request a full build on CircleCI to make sure this integrates well? 
You can do that by putting `[circle full]` in a commit message, so you can make a simple empty commit like this:

```
git commit --allow-empty -m "[circle full] request full build"
```

**Comment by bthirion** [comment]
Indeed, I was going to ask for that ;-)

**Comment by Craskermasker** [comment]
just pushed the empty commit

**Comment by Craskermasker** [comment]
looks like it failed to download some data files

**Comment by NicolasGensollen** [comment]
>looks like it failed to download some data files

Indeed! I restarted the job.

File: examples/04_glm_first_level/write_events_file.py [hunk 1]
```diff
@@ -63,3 +63,8 @@
 tsvfile = 'localizer_events.tsv'
 events.to_csv(tsvfile, sep='\t', index=False)
 print("Created the events file in %s " % tsvfile)
+
+#########################################################################
+# Optionally, the events can be visualized using the plot_event function.
+from nilearn.plotting import plot_event
+plot_event(events)
```
Comment (bthirion) [inline examples/04_glm_first_level/write_events_file.py:70]: End with plt.show()

Comment (Craskermasker) [reply examples/04_glm_first_level/write_events_file.py:71]: added in latest commit

**Comment by bthirion** [comment]
Thx ! It's unrelated to this PR, but the legend does not render well  on the example. Do you see a way to improve that (maybe resizing the figure...)

**Review by NicolasGensollen — COMMENTED** [review]
Could you request a full build? 
It looks like the CI didn't rebuild the example alone for some reason...
LGTM otherwise!

**Comment by Craskermasker** [comment]
done!

**Comment by Craskermasker** [comment]
@NicolasGensollen  - I've merged that fix into this branch now. This is close to the boundary of my git expertise so would appreciate you checking to confirm that it is ok. Thank you!

**Comment by Craskermasker** [comment]
ah it looks like I have inadvertently drawn in a bunch of unrelated commits from another branch. maybe best if I close this and start over...

**Comment by NicolasGensollen** [comment]
No worries. If you wanted to keep this one, you can always rebase in interactive mode and drop all commits except the relevant ones. But I see that you've opened another one so we can leave this one closed.

**Pull request closed without merging**